src/client/index.ts

```diff
@@ -689,6 +750,8 @@ export class Client {
 		}
 	};
 
+	// is it useful? better remove it, we never use it
+	/*
```

Comment:
This is required by the ticket issuer site demo.

Reply:
Better use real user flow for testing to avoid unhandled situations

Reply:
Resolving this conversation. We've added a work around to the examples ticket issuer that fixes the page flickering. This can be later brought into the core lib if we see this as a common design for use of TKN.